Monika After Story/game/script-topics.rpy

```diff
@@ -15825,3 +15833,61 @@ label monika_renewable_energy:
     m 3lksdrb "Sorry, I got carried away there, ahaha!"
     m 1tuu "Debates sure are something, huh?"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_quiet_time",
+            category=['us'],
+            prompt="Do you mind if we don't talk about anything?",
+            pool=True
+        )
+    )
+
+label monika_quiet_time:
+    if mas_isMoniNormal(higher=True):
+        m 3hub "Of course not!"
+        if persistent._mas_pm_social_personality == mas_SP_EXTROVERT:
```

Comment:
Not sure about this path... I'm not an extrovert, but having her assume all extroverts always feel the need to constantly talk feels dubious to me.